CONTRIBUTORS

```diff
@@ -92,3 +92,4 @@ worktycho
 Xenoxis
 xoft (Mattes Dolak/madmaxoft on GH)
 Yeeeeezus (Donated AlchemistVillage prefabs)
+JoeParimucha
```

Comment:
This should be added at the correct alphabetical position, rather than at the end.